db/attachment_test.go

```diff
@@ -10,12 +10,12 @@ package db
 
 import (
 	"context"
+	"crypto/rand"
```

Comment:
This file/`TestLargeAttachments` doesn't seem to _need_ cryptographic randomness? What's the motivation for the change here?

Reply:
https://github.com/golang/go/issues/20661 officially deprecates `math/rand` for `crypto/rand`. The answer seems to be that people are worried that you'll use the wrong one by mistake.

I don't know that we need cryptographic randomness anywhere, but I could see making this mistake. Most tools (goimports, etc) will add cryptographic rand anyway for new code.

We could make an exception for this because I don't think that we currently need cryptographic randomness anywhere, but I don't think this is necessary.

Reply:
Looks like the intention is to switch to a locally scoped `math.Rand.Read()` if you don't need cryptographically secure randomness. We should do that here